src/assets/translations/nl.json

```diff
@@ -25,29 +25,29 @@
         "protectedContent": "Deze beschermde video kan niet worden afgespeeld.",
         "technicalError": "Deze video kan niet worden afgespeeld door een technisch probleem."
     },
-    "exitFullscreen": "Volledig Scherm Verlaten",
+    "exitFullscreen": "Volledig Scherm verlaten",
     "fullscreen": "Fullscreen",
     "hd": "Kwaliteit",
     "liveBroadcast": "Live",
     "logo": "Logo",
-    "mute": "Geluid Dempen",
+    "mute": "Geluid dempen",
     "next": "Volgende",
-    "nextUp": "Volgende Video",
+    "nextUp": "Volgende video",
     "notLive": "Niet live",
     "off": "Uit",
     "pause": "Pause",
     "play": "Start",
     "playback": "Start",
     "playbackRates": "Snelheid",
-    "player": "Video Speler",
+    "player": "Video speler",
```

Comment:
This should be a single compound word, 'videospeler'

Reply:
Agreed